content/pages/kits/lego.md

```diff
@@ -0,0 +1,114 @@
+Title: The LEGO-Based Tables
+Slug: lego
+
+Experimenting with your own LEGO-Based Table is fun and easy!
+
+Already have your table assembled and working? Head over to the [Teaching](/pages/teaching.html) page to learn how to incorporate it into the classroom. Otherwise, this page provides everything you need to know to order, build and operate one yourself.
+
+---
+
+## What are the LEGO-Based Tables?
+
+{% img {static}../../images/kits/lego/technics_table.PNG 400 Technics Table complete setup %}
+
+The LEGO-Based Tables are rotating tank platforms created using LEGOs and other household items for less than $100. The table provides a robust and cheap solution for performing rotating tank experiments that model geophysical fluid dynamics.
+
+---
+
+## New as of October 2020
```

Comment:
## DIYnamics now has TWO Lego-based tables!

Our original LEGO-based table, names simply the **LEGO Table**, remains a great option and remains our most inexpensive and easiest to build platform.  However, especially for repeated or longer demonstrations, we've learned from users all over the world that it can sometimes be inconsistent, or worst of all lose contact or break during a demonstration!  

To address that, we developed an upgrade, beefed up using Lego Technics pieces.  We call this one the **Technics Table**.  It costs a bit more than the Lego Table and has a few more parts, making assembly take a little bit longer.  But we'd say these are well worth it if you plan on using the device many times or are looking for a bit more stability and steady rotation in your demonstrations.